lab-anthony/test/storage-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+
+const Storage = require('../lib/storage');
+const dataDir = new Storage(`${__dirname}/../data`);
+
+describe('Testing the storage module', function() {
+  describe('Testing successful setItem function', function(){
+    it('should write a new file by item id ', function(done){
+      const item = {id:8888};
+      dataDir.setItem('notes', item).then(function(data){
+        expect(data).to.be.equal(item);
+        expect(data.id).to.be.equal(8888);
+        done();
+      }).catch(function(err){
+        console.log(err);
+        expect(err).to.be.equal(undefined);
+        done();
+      });
+    });
+  });
+});
+
+describe('Testing the storage module', function(){
```

Comment:
All of the strings you put into these will get logged in order. The string you'll see on this test will read: 
```shell
Testing the storage module Testing successful fetchItem function should return the requested item
```
but it might be more readable and useful as a message if you had it say something more like
```shell
Testing the storage module fetchItem function should return the requested item
```
or something.